### PR TITLE
G3 2024 v3 #14664 error message for wrong login is fixed

### PR DESCRIPTION
**Pull request by a22ahmom**
Before it wasn't possible to display an error message for wrong login because of Uncaught PDOException that caused a php error for uncaught SyntaxError for invalid JSON. But now when a user enters wrong credentials the error is displayed.

### CODE REVIEW

**Review by a22danwe — CHANGES_REQUESTED** [review]
remove unecessary comments on line 217 in showDugga.php and on line 694 in navheader.php. 

other than that it works, good job!

**Review by a22danwe — APPROVED** [review]
perfecto

File: Shared/navheader.php [hunk 1]
```diff
@@ -691,6 +691,7 @@ function hamburgerToggle() {
 setInterval(
 	function() 
 	{
+		
 		if(gitFetchCooldownSec.innerHTML>0 || gitFetchCooldownMin.innerHTML>0)
```
Comment (a22willi) [inline Shared/navheader.php:695]: This is not a change, remove this file from your PR.

**Review by a22willi — CHANGES_REQUESTED** [review]

**Review by a22willi — APPROVED** [review]
Looks good

**Pull request merged**